test/functional/apps/visualize/_metric_chart.js

```diff
@@ -48,104 +48,104 @@ export default function ({ getService, getPageObjects }) {
         const metricValue = await PageObjects.visualize.getMetric();
-        expect(sumPhpMemory).to.eql(metricValue.split('\n'));
+        expect(sumPhpMemory).to.eql(metricValue);
       });
     });
 
     it('should show Median', async function () {
       const medianBytes = ['5,565.263', '50th percentile of bytes'];
       //  For now, only comparing the text label part of the metric
       log.debug('Aggregation = Median');
-      await PageObjects.visualize.selectAggregation('Median');
+      await PageObjects.visualize.selectAggregation('Median', 'metrics');
       log.debug('Field = bytes');
       await PageObjects.visualize.selectField('bytes', 'metrics');
       await PageObjects.visualize.clickGo();
       await retry.try(async function tryingForTime() {
         const metricValue = await PageObjects.visualize.getMetric();
         // only comparing the text label!
-        expect(medianBytes[1]).to.eql(metricValue.split('\n')[1]);
+        expect(medianBytes[1]).to.eql(metricValue[1]);
       });
     });
 
     it('should show Min', async function () {
-      const minTimestamp = ['September 20th 2015, 00:00:00.000', 'Min @timestamp'];
+      const minTimestamp = ['Sep 20, 2015 @ 00:00:00.000', 'Min @timestamp'];
       log.debug('Aggregation = Min');
-      await PageObjects.visualize.selectAggregation('Min');
+      await PageObjects.visualize.selectAggregation('Min', 'metrics');
       log.debug('Field = @timestamp');
       await PageObjects.visualize.selectField('@timestamp', 'metrics');
       await PageObjects.visualize.clickGo();
       await retry.try(async function tryingForTime() {
         const metricValue = await PageObjects.visualize.getMetric();
-        expect(minTimestamp).to.eql(metricValue.split('\n'));
+        expect(minTimestamp).to.eql(metricValue);
       });
     });
 
     it('should show Max', async function () {
-      const maxRelatedContentArticleModifiedTime = ['April 4th 2015, 00:54:41.000', 'Max relatedContent.article:modified_time'];
+      const maxRelatedContentArticleModifiedTime = ['Apr 4, 2015 @ 00:54:41.000', 'Max relatedContent.article:modified_time'];
       log.debug('Aggregation = Max');
-      await PageObjects.visualize.selectAggregation('Max');
+      await PageObjects.visualize.selectAggregation('Max', 'metrics');
       log.debug('Field = relatedContent.article:modified_time');
       await PageObjects.visualize.selectField('relatedContent.article:modified_time', 'metrics');
       await PageObjects.visualize.clickGo();
       await retry.try(async function tryingForTime() {
         const metricValue = await PageObjects.visualize.getMetric();
-        expect(maxRelatedContentArticleModifiedTime).to.eql(metricValue.split('\n'));
+        expect(maxRelatedContentArticleModifiedTime).to.eql(metricValue);
       });
     });
 
-    it('should show Standard Deviation', async function () {
+    it.skip('should show Standard Deviation', async function () {
```

Comment:
`Standard Deviation` aggregation is not available for Metric (https://github.com/elastic/kibana/blob/master/src/legacy/core_plugins/metric_vis/public/metric_vis.js#L97). 
@timroes @dmlemeshko can this test be removed?

Reply:
I removed this obsolete test.